File: 0001_Two_Sum/go/naive.go
// O(n^2): Try every possible combination of sums in the array
func twoSum(nums []int, target int) []int {
    for index1, number1 := range nums {
        for index2, number2 := range nums {
            if index1 == index2 {
                continue
            } else if (number1 + number2) == target {
                return []int{index1, index2}
            }
        }
    }
    return []int{0, 0}
}

File: 0219_Contains_Duplicate_II/python3/naive.py
class Solution:
    def containsNearbyDuplicate(self, nums: List[int], k: int) -> bool:
        n = len(nums)
        for i in range(n):
            for j in range(i+1, n):
                if nums[i] == nums[j] and abs(i-j) <= k:
                    return True
        return False
